### PR TITLE
Style pass on Tasks, Components, and Apps

### PR DESCRIPTION
**Pull request by briansage**
The Tasks view header lacked a few styles. This commit styles the
header of the view to match other views.

App and Component names were inconsistent on different screens in
the UX lifecycle of creation and administration. This commit makes them
more uniform across different views.

### CODE REVIEW

File: lib/instances/InstancesSummary.component.js [hunk 1]
```diff
@@ -23,13 +23,34 @@ export default class InstancesSummary extends React.Component {
     const running = instances
       .filter(instance => instance.get('status') !== 'STOPPED')
 
+    const UIAddSubtract = props =>{
```
Comment (esmevane) [inline lib/instances/InstancesSummary.component.js]: Could this get extracted to a standalone component? Ideally with tests, though I'm OK pairing on that or doing it in a future commit if you want

**Comment by esmevane** [comment]
Looks good - it'd be nice to get the plus / minus into a separate file but _not necessary right now_, that can be done as a future commit if that's not something you can get to for a bit.

File: lib/instances/InstancesSummary.component.js (hunk 1 above)
Comment (briansage) [inline lib/instances/InstancesSummary.component.js]: OK, OK, you caught me. I just extracted this, but, TBH, I'm not really happy with the rudimentary markup tests I added for it. Would love to pair on this to test the click behavior of the element.

**Comment by esmevane** [comment]
Merged

**Pull request merged**